week1/homework/src/server.js

```diff
@@ -5,13 +5,38 @@ const http = require('http');
 /* `createServer` MUST return an instance of `http.Server` otherwise the tests
  * will fail.
  */
+
 function createServer(port) {
   let state = 10;
 
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
-  });
+    if (request.url.startsWith('/state')) {
```

Comment:
Any specific reason why you went with `startsWith` method here but used triple equals in other cases?

Reply:
Oh sorry, I was trying to do something different then I forgt to change it back.